Plugin.cpp

```diff
@@ -24,6 +24,18 @@ BINARYNINJAPLUGIN bool CorePluginInit()
     Workflow::registerActivities();
     Commands::registerCommands();
 
+    BinaryNinja::Ref<BinaryNinja::Settings> settings = BinaryNinja::Settings::Instance();
+    settings->RegisterGroup("objc", "Objective-C");
+
+    settings->RegisterSetting("objc.cleanupARCCode",
+	R"({
+	"title" : "ARC Cleanup",
+	"type" : "boolean",
+	"default" : true,
+	"description" : "Remove ARC related code, i.e. calls to _objc_release, _objc_retain, and other ARC functions, from ILs"
```

Comment:
Title should be "Hide ARC Calls". 
- "ARC related" should be hyphenated
- "i.e." should be "e.g."
- ", from ILs" should be " in ILs"